vendor/licenses/git_submodule/mdp.yml

```diff
@@ -0,0 +1,492 @@
+---
```

Comment:
This is not a valid license file. Please see the CONTRIBUTING.md file.

Reply:
✅ Done, Added MIT 2023 and Apache 2.0